pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdTopicPolicies.java

```diff
@@ -85,6 +85,10 @@ public CmdTopicPolicies(Supplier<PulsarAdmin> admin) {
         jcommander.addCommand("get-subscription-dispatch-rate", new GetSubscriptionDispatchRate());
         jcommander.addCommand("set-subscription-dispatch-rate", new SetSubscriptionDispatchRate());
         jcommander.addCommand("remove-subscription-dispatch-rate", new RemoveSubscriptionDispatchRate());
+
+        jcommander.addCommand("get-replicator-dispatch-rate",new GetReplicatorDispatchRate());
```

Comment:
A space is required after the comma, from L89 to L91

Reply:
fixed